addon/components/ember-notify/message.js

```diff
@@ -2,6 +2,10 @@ import Ember from 'ember';
 import layout from '../../templates/components/ember-notify/message';
 import Notify from 'ember-notify';
 
+function checkSendCloseIntent() {
+  if (this.get('isDestroyed')) return;
```

Comment:
That doesn't make sense. If `isDestroyed` is true here then it will also be true when `closeIntent` is fired.